node_helper.js

```diff
@@ -60,12 +60,13 @@ module.exports = NodeHelper.create({
         let date = new Date();
         date.setDate(date.getDate() - this.config.daysInPast);
         const startDate = date.toISOString().slice(0, 10);
-        date.setDate(date.getDate() + this.config.daysInPast + this.config.daysAhead);
+        date.setDate(date.getDate() + this.config.daysAhead);
         const endDate = date.toISOString().slice(0, 10);
 
         const query = qs.stringify({startDate, endDate, expand: 'schedule.linescore'});
-
-        const response = await fetch(`${BASE_URL}/schedule?${query}`);
+        const url = `${BASE_URL}/schedule?${query}`;
+        console.log(`MMM-NHL Fetching:${BASE_URL}/schedule?${query}`);
```

Comment:
I think the log will not be necessary for running it in prod, probably a leftover from investigating the extra day issue you described.